tools/c7n_left/tests/test_left.py

```diff
@@ -877,6 +881,30 @@ def test_cli_no_policies(tmp_path, caplog):
     assert caplog.record_tuples == [("c7n.iac", 30, "no policies found")]
 
 
+def test_cli_output_rich_mod_resource_ref(tmp_path, debug_cli_runner):
```

Comment:
this should probably get a pytest.skipif not GITHUB_ACTION due to requiring network access.